### PR TITLE
docs: Integrate jinno for demo

### PR DESCRIPTION
**Pull request by morZloof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,36 +141,36 @@ Otherwise, you can simply import along with the backend itself (as shown above).
...
-|[`allowUnique`](#allowUnique) | `Boolean` | `true` | Boolean value to control whether tags should be unqiue
-|[`allowDragDrop`](#allowDragDrop) | `Boolean` | `true` | Boolean value to control whether tags should have drag-n-drop features enabled
-|[`renderSuggestion`](#renderSuggestion) | `Function` | `undefined` | Render prop for rendering your own suggestions
+|[`allowUnique`](#allowUnique) | `Boolean` | `true` | Boolean value to control whether tags should be unqiue [Demo](http://www.jinno.io/app/6/allowUnique?source=react-tag-input)
+|[`allowDragDrop`](#allowDragDrop) | `Boolean` | `true` | Boolean value to control whether tags should have drag-n-drop features enabled [Demo](http://www.jinno.io/app/6/allowDragDrop?source=react-tag-input)
+|[`renderSuggestion`](#renderSuggestion) | `Function` | `undefined` | Render prop for rendering your own suggestions [Demo](http://www.jinno.io/app/6/renderSuggestion?source=react-tag-input)
 | [`inputProps`](#inputProps) | Object |`{}` | The extra attributes which are passed to the input field.
 
-### tags
+### tags [Demo](http://www.jinno.io/app/6/tags?source=react-tag-input)
```
Comment (ad1992) [inline README.md:173]:
```suggestion
### tags 
Check out the [Demo](http://www.jinno.io/app/6/tags?source=react-tag-input)
```
The demo along with header tag looks pretty big, lets add it to the next line for all as shown above

Comment (ad1992) [reply README.md:173]: Also demo link for [`delimeters`](https://github.com/react-tags/react-tags/blob/965e654b8b4c50cde31a92f7a23e980c15f53250/README.md#delimiters) is missing

File: README.md [hunk 2]
```diff
@@ -141,36 +141,36 @@ Otherwise, you can simply import along with the backend itself (as shown above).
 
 Option | Type | Default | Description
 --- | --- | --- | ---
-|[`tags`](#tags) | `Array` | `[]` | An array of tags that are displayed as pre-selected
-|[`suggestions`](#suggestions) | `Array` | `[]` | An array of suggestions that are used as basis for showing suggestions
+|[`tags`](#tags) | `Array` | `[]` | An array of tags that are displayed as pre-selected [Demo](http://www.jinno.io/app/6/tags?source=react-tag-input)
```
Comment (ad1992) [inline README.md:144]:
```suggestion
|[`tags`](#tags) | `Array` | `[]` | An array of tags that are displayed as pre-selected.
 Check out the  [Demo](http://www.jinno.io/app/6/tags?source=react-tag-input)
```
Lets append the demo in a new line for all the attributes so its more clear

**Review by ad1992 — COMMENTED** [review]
Thanks a lot @morZloof for the contribution, could you make the suggested changes and we can merge then

File: README.md [hunk 3]
```diff
@@ -21,10 +21,10 @@ Follow [@ReactTags](https://twitter.com/ReactTags) on twitter!
 Because I was looking for an excuse to build a standalone component and publish it in the wild? To be honest, I needed a tagging component that provided the above features for my [React-Surveyman](http://github.com/prakhar1989/react-surveyman) project. Since I was unable to find one which met my requirements (and the fact that I generally enjoy re-inventing the wheel) this is what I came up with.
 
 
-## Demo
+## demo
```
Comment (ad1992) [inline README.md:24]:
```suggestion
## Demo
```

**Comment by ad1992** [comment]
@morZloof I pushed some fixes, let me know if this looks good then will merge

**Comment by ad1992** [comment]
Also @morZloof I can see there are some attributes missing ex `delimiters`, could you please add those as well jinno so it works. Everywhere where we are adding demo links should work else it can confuse the users, in case there is some which might not work now I can remove those, let me know.

**Comment by morZloof** [comment]
> Also @morZloof I can see there are some attributes missing ex `delimiters`, could you please add those as well in jinno so it works. Everywhere where we are adding demo links should work else it can confuse the users, in case there is some which might not work now I can remove those, let me know.

File: README.md [hunk 4]
```diff
@@ -211,14 +213,15 @@ const Keys = {
     delimiters={[Keys.TAB, Keys.SPACE, Keys.COMMA]}
  />
 ```
-
+Here is a [demo](http://www.jinno.io/app/6/delimiters?source=react-tag-input).
```
Comment (ad1992) [inline README.md:216]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -141,36 +141,36 @@ Otherwise, you can simply import along with the backend itself (as shown above).
...
-|[`name`](#name) | `String` | `undefined` | The `name` attribute added to the input
-|[`id`](#id) | `String` | `undefined` | The `id` attribute added to the input
-|[`maxLength`](#maxLength) | `Number` | `Infinity` | The `maxLength` attribute added to the input
-|[`inline`](#inline) | `Boolean` | `true` | Render input field and selected tags in-line
+|[`autocomplete`](#autocomplete) | `Boolean`/`Number` | `false` | Ensure the first matching suggestion is automatically converted to a tag when a [delimiter](#delimiters) key is pressed. Here is a [demo](http://www.jinno.io/app/6/autocomplete?source=react-tag-input).
+|[`readOnly`](#readOnly) | `Boolean` | `false` | Read-only mode without the input box and `removeComponent` and drag-n-drop features disabled. Here is a [demo](http://www.jinno.io/app/6/readOnly?source=react-tag-input).
+|[`name`](#name) | `String` | `undefined` | The `name` attribute added to the input . Here is a [demo](http://www.jinno.io/app/6/name?source=react-tag-input).
+|[`id`](#id) | `String` | `undefined` | The `id` attribute added to the input . Here is a[demo](http://www.jinno.io/app/6/id?source=react-tag-input).
```
Comment (ad1992) [inline README.md:164]:
```suggestion
|[`id`](#id) | `String` | `undefined` | The `id` attribute added to the input . Here is a [demo](http://www.jinno.io/app/6/id?source=react-tag-input).
```

**Review by ad1992 — APPROVED** [review]

**Comment by ad1992** [comment]
Thanks for the contribution @morZloof , merging 🎉

**Pull request merged**